opt/scopeserver/dataserver/utils/search_space.py

```diff
@@ -182,3 +184,53 @@ def add_metrics(self) -> None:
         for metric in self.meta_data["metrics"]:
             metrics.append(metric["name"])
         self.add_elements(elements=metrics, element_type="metric")
+
+
+def load_ss(loom) -> SearchSpace:
```

Comment:
Maybe just `load` now?

Reply:
Does the `SearchSpace` _really_ need to know about the loom? The loom should know about the search space but not the other way around?